cog/slots.py

```diff
@@ -91,22 +93,30 @@ def _assign_reels(self, reels):
     async def _roll_reels(self, reels, msg, ctx):
         ''' Roll each reel for a certain amount of ticks '''
         bottom_slot0, bottom_slot1, bottom_slot2 = 2, randint(0, _SYMBOL_AMOUNT_IN_REEL - 4), randint(0, _SYMBOL_AMOUNT_IN_REEL - 4)
-        reel0_ticks = randint(5, 10)
-        reel1_ticks = reel0_ticks + randint(1, 3)
-        reel2_ticks = reel1_ticks + randint(1, 3) # reel2_ticks always have the largest amount of ticks
+        reel0_ticks = randint(1, 2)
+        reel1_ticks = reel0_ticks + randint(0,1)
+        reel2_ticks = reel1_ticks + randint(0,1)
         content_string = ""
         
         reel0_border = "**:**"
         reel1_border = "**:**"
         reel2_border = "**:**"
 
+        slot_0 = None
+        slot_1 = None
+        slot_2 = None
+
         while(reel2_ticks >= 0):
             content_string = ""
+            content_string += "--CIRNO  SLOTS--\n"
```

Comment:
String concatenation is considered inefficient. What you should is give these separate variable names and at the end you do a '\n'.join(). It's considerably faster and more modular. This should be done when you have a lot of string concats. I didn't learn this until after I did it, but letting you know now so you can also fix it when you polish.